playbooks/site.yml

```diff
@@ -17,27 +17,31 @@
   tags:
     - maas
 
-- include: maas-poller-all.yml
-  tags:
-    - maas
+# - include: maas-poller-all.yml
```

Comment:
Question here:  why all these other playbooks disabled?  Should we make a different branch/release for K8?

Reply:
I commented them out to speed up testing...